packages/suite-data/src/message-system/constants/index.ts

```diff
@@ -21,14 +22,10 @@ export const PACKAGES_ROOT = resolve(PROJECT_ROOT, '../../..');
 export const SCHEMA_PATH = join(PROJECT_ROOT, 'schema', SCHEMA_FILENAME);
 export const CONFIG_PATH = join(PROJECT_ROOT, 'config', CONFIG_FILENAME);
 
-const DEV_JWS_PRIVATE_KEY = `-----BEGIN EC PRIVATE KEY-----
+// use key from JWS_PRIVATE_KEY_FILE or use the default dev private key
```

Comment:
@tomasklim What about
```suggestion
// The only protected branch is the `codesign` branch which is the only branch used to run code signing builds.
// Use production keys for code signing builds and development keys for all other builds.
```